### DISCUSSION TITLE
开机启动会自动弹出界面

### DISCUSSION THREAD

**Discussion by chansee97** [he3-app/he3-feedback#154] [Bug Report]
我设置了开机启动，但是现在每次开机都会弹出一次界面，然后我再手动关闭，这是一个bug吗？我该如何修改这个行为？

**Comment by lyzhang1999** [comment]
感谢你的反馈，我已经记录了该问题，并建立了 issues 来跟踪：https://github.com/he3-app/he3-feedback/issues/155 ， 您可以关注该 issues 以获得修复进度。
